docs/develop_tools/web_ui.md

```diff
@@ -0,0 +1,84 @@
+<!--
+- Licensed to the Apache Software Foundation (ASF) under one or more
+- contributor license agreements.  See the NOTICE file distributed with
+- this work for additional information regarding copyright ownership.
+- The ASF licenses this file to You under the Apache License, Version 2.0
+- (the "License"); you may not use this file except in compliance with
+- the License.  You may obtain a copy of the License at
+-
+-   http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-->
+
+# Develop Kyuubi Web UI
+
+Requirement:
+
+1. Fork Kyuubi source code[version >= 1.8.0]
+2. Install [nodejs](https://nodejs.org/en/)[Recommend 16.x.x]
+
+## Install Dependencies
+
+```shell
+cd ${KYUUBI_SOURCE_CODE}/kyuubi-server/web-ui
+npm install
+```
+
+## Develop
+
+Notice:
+
+Before you start the Web UI project, please make sure the Kyuubi server has been started.
+
+Kyuubi Web UI will proxy the requests to Kyuubi server,  with the default endpoint path to`http://localhost:10099`. Modify `VITE_APP_DEV_WEB_URL` in `.env.development` for customizing targeted endpoint path.
+
+#### Why proxy to http://localhost:10099
+
+Current kyuubi server binds on `http://0.0.0.0:10099` in case you are running kyuubi server in MacOS or Windows(If in linux, you should config kyuubi server `kyuubi.frontend.rest.bind.host=0.0.0.0`, or change `VITE_APP_DEV_WEB_URL` in `.env.development`).
```

Comment:
capitalize the words "Kyuubi" "Linux" as "Windows" does. And "macOS", not "MacOS"